CHANGELOG.md

```diff
@@ -8,8 +7,18 @@
 
 ### Features
 
+* **Adds the embedding module to be able to embed Elements**
+  * Problem: Many NLP applications require the ability to represent parts of documents in a semantic way. Until now, Unstructured did not have text embedding ability within the core library.
+  * Feature: This embedding module is able to track embeddings related data with a class, embed a list of elements, and return an updated list of Elements with the *embeddings* property. The module is also able to embed query strings.
+  * Importance: Ability to embed documents, or parts of documents will enable users to make use of these semantic representations in different NLP applications, such as search, retrieval, and retrieval augmented generation.
+
```

Comment:
appreciate adding the importance here. probably should follow the single paragraph format of all of the other recent changelog items